<?xml version="1.0"?>

<launch>
    <include file="$(find final_assignment)/launch/simulation_gmapping.launch"/>
    <include file="$(find final_assignment)/launch/move_base.launch"/>
    <node pkg="final_assignment" type="menu.py" name="menu" output="screen" required="true" launch-prefix="xterm -e"/>
    <node pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" name="teleop" output="screen" launch-prefix="xterm -e">
    <remap from="cmd_vel" to="input_cmd_vel"/>
    
    </node>
</launch>
